src/commands/mocks/spk-config.yaml

```diff
@@ -6,11 +6,11 @@ azure_cli:
 
 # Azure DevOps Configuration
 azure_devops: # Should this live under the `spk` configuration?
-  org: "my-az-do-org"
-  project: "my-mono-repo-project"
-  hld_repository: "my-hld-repository"
-  manifest_repository: "my-manifest-repository"
-  access_token: "access-token" # This is your Personal Access Token with permission to modify and access this private repo. Ex. "${env:AZDO_PERSONAL_ACCESS_TOKEN}". Leave this empty if project is public
+  access_token: "hpe3a9oiswgcodtfdpzfiek3saxbrh5if1fp673xihgc5ap467a" # This is your Personal Access Token with permission to modify and access this private repo. Leave this empty if project is public
+  hld_repository: "dev.azure.com/bhnook/fabrikam/_git/hld" # Repository URL for your Bedrock HLDs
+  manifest_repository: "dev.azure.com/bhnook/fabrikam/_git/materialized" # Repository URL that is configured for flux. This holds the kubernetes manifests that is generated by fabrikate.
+  org: "https://dev.azure.com/bhnook" # Your AzDo Org
```

Comment:
@evanlouie @mtarng we should just use `bhnook` here

Reply:
yes, specifying a URL in this one is going to break introspection